src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/command/ValidateYamlCommand.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.em.orchestrator.automatedbundling.command;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.em.orchestrator.automatedbundling.configuration.LocalConfigurationLoader;
+
+import java.io.File;
+import java.net.URL;
+
+@Service
+public class ValidateYamlCommand {
+
+    @Autowired
+    private LocalConfigurationLoader loader;
+
+    private static Logger log = LoggerFactory.getLogger(ValidateYamlCommand.class);
+
+    public ValidateYamlCommand(LocalConfigurationLoader loader) {
+        this.loader = loader;
+    }
+
+    public boolean run() {
+        URL url = Thread
+            .currentThread()
+            .getContextClassLoader()
+            .getResource("bundleconfiguration/");
```

Comment:
will this always work? what if bundleconfiguration hasn't been assigned a value?

Reply:
it's referencing the folder inside `main/resources/`

Reply:
ah cool. There are multiple - which one is it gonna look at?

Reply:
all of them, it prints something like:

```
2019-07-10T14:23:26.574 INFO  [main] u.g.h.r.e.o.a.command.ValidateYamlCommand Validating example.yaml succeeded
2019-07-10T14:23:26.575 INFO  [main] u.g.h.r.e.o.a.command.ValidateYamlCommand Validating example-with-filename.yaml succeeded
2019-07-10T14:23:26.575 INFO  [main] u.g.h.r.e.o.a.command.ValidateYamlCommand 0 error(s) found```